src/Microsoft.AspNetCore.BuildTools.ApiCheck/Microsoft.AspNetCore.BuildTools.ApiCheck.nuspec

```diff
@@ -6,13 +6,12 @@
     <authors>Microsoft</authors>
     <description>Generates API baselines and detects breaking changes in APIs.</description>
     <dependencies>
-      <group targetFramework=".NETStandard1.0" />
+      <group targetFramework=".NETStandard2.0" />
```

Comment:
This will prevent us from using API Check on anything that is not netstandard2.0 compatible. Recommend keeping this as ns1.0.